packaging/src/docker/README.md

```diff
@@ -117,7 +117,7 @@ export POSTGRES_LOCAL_PATH=your_local_path_to_postgres_driver
 ```
 Example:
 ```shell
-mvn dependency:copy -Dartifact="org.postgresql:postgresql:42.5.1" && \
+mvn dependency:copy -Dartifact="org.postgresql:postgresql:42.7.3" && \
```

Comment:
Could you also please fix the version in these lines? Thanks.
https://github.com/apache/hive/blob/1a969f6642d4081027040f97e6a689bf2e687db3/packaging/src/docker/README.md?plain=1#L63
https://github.com/apache/hive/blob/1a969f6642d4081027040f97e6a689bf2e687db3/packaging/src/docker/README.md?plain=1#L72
https://github.com/apache/hive/blob/1a969f6642d4081027040f97e6a689bf2e687db3/packaging/src/docker/README.md?plain=1#L121